modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/ExecutorImpl.scala

```diff
@@ -80,6 +113,10 @@ class ExecutorImpl private (cancelRef: TaskRef[Set[SPObservationID]], stateRef:
       })
   }
 
+  // Important, make it a def so each client gets a new one
+  def sequenceEvents: Process[Task, SeqexecEvent] = Process.emit(SeqexecConnectionOpenEvent) ++ eventsQueue.subscribe
```

Comment:
This the rest of the world listen for events generated inside the execution engine